### PR TITLE
fix signin issue

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

There was a collision between some tools that we use and the PEM file we need to log on with.

## Security Considerations

N/A

### CODE REVIEW

File: app/main/views/sign_in.py [hunk 1]
```diff
@@ -27,22 +27,12 @@
...
-            f"-----BEGIN {private_key}-----",
-            new_keystring,
-            f"-----END {private_key}-----",
-        ]
-    )
-    new_keystring = f"{new_keystring}\n"
+    arr = orig.split("-----")
+    begin = arr[1]
+    end = arr[3]
+    middle = arr[2].strip()
```
Comment (jskinne3) [inline app/main/views/sign_in.py:35]: Don't you also need to add newlines?

**Review by jskinne3 — CHANGES_REQUESTED** [review]

File: app/main/views/sign_in.py (hunk 1 above)
Comment (terrazoon) [reply app/main/views/sign_in.py:35]: I do add new lines, see:


    new_keystring = f"-----{begin}-----\n{middle}\n-----{end}-----\n"

Comment (jskinne3) [reply app/main/views/sign_in.py:35]: Oh there aren't new lines needed _within_ the middle text? OK

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Pull request merged**